lib/pf/Portal/Profile.pm

```diff
@@ -69,7 +69,7 @@ sub new {
 
```

Comment:
I was expecting this file to be renamed as well to lib/pf/Connection/Profile.pm and the class name to be changed